src/coreclr/jit/gentree.cpp

```diff
@@ -26863,6 +26961,11 @@ bool GenTreeHWIntrinsic::OperIsEmbRoundingEnabled() const
         case NI_AVX512F_MultiplyScalar:
         case NI_AVX512F_SubtractScalar:
         case NI_AVX512F_SqrtScalar:
+        case NI_AVX10v1_AddScalar:
```

Comment:
Looks like we're missing logic further up in `OperIsEmbBroadcastCompatible` where `NI_AVX512F_VL_ConvertToVector128UInt32` should be handled as `NI_AVX10v1_ConvertToVector128UInt32`

Reply:
Looks like a new change due to merge with main. I have added the handling as you suggested.